extensions/cpsection/aboutcomputer/model.py

```diff
@@ -178,6 +178,8 @@ def get_wireless_firmware():
             continue
 
         try:
+            driver = None
+            version = None
```

Comment:
Consider how `None` will be processed in later code; if _driver_ was set to _libertas_, then _version.startswith_ would fail.